src/DynamoManipulation/MousePointManipulator.cs

```diff
@@ -290,53 +290,4 @@ private static void ModifyInputNode(NodeModel inputNode, double amount)
 
         #endregion
     }
-
-    internal static class PointExtensions
```

Comment:
moved classes to `NodeManipulator.cs` as these classes can be potentially used by all `NodeManipulator` subclasses